roles/aks-setup/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: "Create resource group digitransit-aks-{{ az_environment }}"
```

Comment:
Maybe we should add -rg at the end of resource group names

Reply:
Also seems like currently the stuff if azure are named digitransit-env-resource instead of digitransit-resource-env so maybe we should stick to that